src/txdb.h

```diff
@@ -6,6 +6,7 @@
 #ifndef BITCOIN_TXDB_H
 #define BITCOIN_TXDB_H
 
+#include "addressindex.h"
```

Comment:
I wonder if this include should go under a conditionally compiled section since it is used by the Address Indexing only.

```suggestion
#ifdef ENABLE_ADDRESS_INDEXING
#include "addressindex.h"
#endif // ENABLE_ADDRESS_INDEXING
```

Reply:
Done (added forward declaration for AddressType)